handlers/validators.go

```diff
@@ -365,12 +421,26 @@ func ValidatorsData(w http.ResponseWriter, r *http.Request) {
 		tableData[i] = append(tableData[i], html.EscapeString(v.Name))
 	}
 
+	var totalValCounts []*counts
+	qry = "SELECT count(*) as total FROM validators"
+	err = db.DB.Select(&totalValCounts, qry)
+	if err != nil {
+		logger.Errorf("error retrieving validators total count: %v", err)
+		http.Error(w, "Internal server error", 503)
+		return
+	}
+
 	data := &types.DataTableResponse{
 		Draw:            dataQuery.Draw,
-		RecordsTotal:    totalCount,
-		RecordsFiltered: filteredCount,
+		RecordsTotal:    totalValCounts[0].Total,
+		RecordsFiltered: 0,
 		Data:            tableData,
 	}
+	if isAll {
+		data.RecordsFiltered = totalValCounts[0].Total
+	} else if validators != nil {
```

Comment:
please also check that the array has a length > 0 before you access an element at position 0

Reply:
Did this change here: https://github.com/gobitfly/eth2-beaconchain-explorer/pull/1266